backend/internal/core/currencies/currencies.json

```
@@ -131,6 +131,13 @@
     "symbol": "B$",
     "name": "Bahamian Dollar"
   },
+  {
+    "code": "BTC",
+    "local": "Bitcoin",
+    "symbol": "₿",
+    "name": "Bitcoin"
+  },
+  {
```

It seems you've added an extra curly brace
```suggestion
  {
    "code": "BTC",
    "local": "Bitcoin",
    "symbol": "₿",
    "name": "Bitcoin"
  },
```